packages/desktop/components/popup/popups/CheckForUpdatesPopup.svelte

```diff
@@ -17,73 +17,81 @@
     }
 
     function onVisitDownloadsClick(): void {
-        openUrlInBrowser('https://firefly.iota.org')
+        openUrlInBrowser('https://bloomwallet.io/')
     }
 
     function onCloseClick(): void {
         closePopup()
     }
 
+    let continueButton: ButtonProps
+    function setContinueButton(): void {
```

Comment:
What about rewriting it the following way? Like this we dont need to call it in the onmount

```suggestion
    let continueButton: ButtonProps = getContinueButton()
    function getContinueButton(): ButtonProps {
```